vite.config.js

```diff
@@ -2,40 +2,34 @@ import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 import visualizer from 'rollup-plugin-visualizer';
 import { resolve } from "path";
-import { nodeResolve } from '@rollup/plugin-node-resolve'; 
-
-// Workaround for this open solid-client-authn bug:
-// https://github.com/inrupt/solid-client-authn-js/issues/3001
-// Once that issue is fixed and the solid-client-authn-js version is bumped
-// We can delete this function and all related code
-function withCustomResolver(
```

Comment:
Inrupt's V2 libraries now support ES Modules out of the box, so this custom resolver is no longer needed.